pkg/virt-handler/device-manager/deviceplugin/v1alpha/BUILD

```diff
@@ -0,0 +1,41 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
I just curious why do we have all this files(`pkg/virt-handler/device-manager/deviceplugin`), can we just import them without copy paste?

Reply:
That pulled in the entire kubernetes tree as a dependency, which means we would have had 9500+ files added to the vendor tree for no good reason. I assume this will be sorted out upstream at some point as it affects all consumers of device plugins.